crates/optimism/evm/src/l1.rs

```diff
@@ -342,7 +347,7 @@ where
 
         // Commit the create2 deployer account to the database.
         db.commit(HashMap::from_iter([(CREATE_2_DEPLOYER_ADDR, revm_acc)]));
-        return Ok(())
+        return Ok(());
```

Comment:
please revert unrelated formatting change. make sure to only use nightly toolchain for `fmt` in reth.